### PR TITLE
Add manpages for clr_power and clr-power-tweaks.conf

### PR DESCRIPTION
**Pull request by puneetse**
This PR Adds manpage content for clr_power and clr-power-tweaks.conf and moves docs into a man subfolder.

### CODE REVIEW

**Comment by puneetse** [comment]
@ahkok ready for review.

File: man/clr-power-tweaks.conf.5.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+**`clr-power-tweaks.conf`** is an optional configuration file for the
+clr_power(1) utility. The configuration file contains kernel and device
+parameter values used by the clr_power(1) utility. 
+
+Configuration parameter values placed in **`clr-power-tweaks.conf`** will
+supersede the built-in defaults configured by clr_power(1). New parameters can
+be added to the configuration to supplement the built-in values. Built-in
+parameters can be changed by defining them in **`clr-power-tweaks.conf`** with
+a different value.
+
```
Comment (ahkok) [inline man/clr-power-tweaks.conf.5.md:20]: This is missing the "don't touch" option that it has. If there is no value, then clr_power will not touch the file even if it has a builtin value for it. E.g. /proc/sys/vm/swappiness I believe is set to something like `30` by the kernel default. clr-power changes it to `10`. You can disable clr-power from touching the file entirely by listing the file without a value.

File: man/clr-power-tweaks.conf.5.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor powersave
+   ```
+2. **`clr-power-tweaks.conf`** entry to add two new parameters to increase UDP
+   buffers All other values built-in to clr_power(1) would still get applied. 
+
+   ```
+   /proc/sys/net/ipv4/udp_rmem_min 8192
+   /proc/sys/net/ipv4/udp_wmem_min 8192
+   ```
+
```
Comment (ahkok) [inline man/clr-power-tweaks.conf.5.md:64]: We'd want an example of the "no touch" here as well.

**Comment by ahkok** [comment]
One thing missing but otherwise, great!

File: man/clr-power-tweaks.conf.5.md (hunk 1 above)
Comment (ahkok) [reply man/clr-power-tweaks.conf.5.md:20]: This is still an open - you have "supersede" and "add" but not described the "prevent modification of parameters" covered.

Comment (puneetse) [reply man/clr-power-tweaks.conf.5.md:20]: I added it to the FILE FORMAT section:

> Any parameter listed in the configuration file without a value will be excluded by clr_power(1) and the parameter will inherit the kernel's built-in default parameter value.

but now that I'm looking at it again, it makes more sense here.

**Review by ahkok — APPROVED** [review]

**Comment by ahkok** [comment]
thumbs up, thanks!

**Pull request merged**